nodes/nodes/ecosystems/eco-node-health-check-operator.adoc

```diff
@@ -25,9 +25,9 @@ include::modules/eco-node-health-check-operator-creating-node-health-check.adoc[
 
 [id="gather-data-nhc"]
 == Gathering data about the Node Health Check Operator
-To collect debugging information about the Node Health Check Operator, use the `must-gather` tool. For information about the `must-gather` image for the Node Health Check Operator, see xref:../../support/gathering-cluster-data.adoc#gathering-data-specific-features_gathering-cluster-data[Gathering data about specific features].
+To collect debugging information about the Node Health Check Operator, use the `must-gather` tool. For information about the `must-gather` image for the Node Health Check Operator, see xref:../../../support/gathering-cluster-data.adoc#gathering-data-specific-features_gathering-cluster-data[Gathering data about specific features].
 
 [id="additional-resources-nhc-operator-installation"]
 == Additional resources
-* xref:../../operators/admin/olm-upgrading-operators.adoc#olm-changing-update-channel_olm-upgrading-operators[Changing the update channel for an Operator]
-* The Node Health Check Operator is supported in a restricted network environment. For more information, see xref:../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks].
+* xref:../../../operators/admin/olm-upgrading-operators.adoc#olm-changing-update-channel_olm-upgrading-operators[Changing the update channel for an Operator]
+* The Node Health Check Operator is supported in a restricted network environment. For more information, see xref:../../../operators/admin/olm-restricted-networks.adoc#olm-restricted-networks[Using Operator Lifecycle Manager on restricted networks].
```

Comment:
I struggle getting around this myself, but Additional resources can only contain a list of bulleted links, no text. IMO, I think the link text you use stands on its own just fine.

[1] https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=JUP&title=Jupiter+Source+files+readiness+Checklist#JupiterSourcefilesreadinessChecklist-Additionalresourcesspecificrequirements

Reply:
Thanks @mjpytlak I've applied your feedback.